index.rst

```diff
@@ -126,12 +126,12 @@ Supported Microcontrollers
 
 .. imgtable::
 
-    ESP32, components/esp32, esp32.svg
-    ESP8266, components/esp8266, esp8266.svg
-    RP2040, components/rp2040, rp2040.svg
-    BK72xx, components/libretiny, bk72xx.svg
-    RTL87xx, components/libretiny, rtl87xx.svg
-    host, components/host, host.svg, dark-invert
+    ESP32, components/esp32, esp32.svg, dark-invert
+    ESP8266, components/esp8266, esp8266.svg, dark-invert
+    RP2040, components/rp2040, rp2040.svg, dark-invert
+    BK72xx, components/libretiny, bk72xx.svg, dark-invert
+    RTL87xx, components/libretiny, rtl87xx.svg, dark-invert
```

Comment:
Not sure I like having the generated images inverted...

Reply:
😎